hypothesis-python/src/hypothesis/internal/escalation.py

```diff
@@ -28,6 +28,9 @@
 from hypothesis.internal.compat import text_type, binary_type, \
     encoded_filepath
 
+if False:
+    from typing import Dict  # noqa
```

Comment:
If we need `#noqa` on all our type imports that's going to get old *fast*. Can we add a general ignore line in config somewhere? (I'm not sure how flake8 config works for this)

Reply:
You can for `coverage`, but I can't find such an option for `flake8` :-(

Reply:
And https://gitlab.com/pycqa/flake8/issues/118 doesn't look likely to see any movement in the near future. ☹️